### ISSUE TITLE
Reactive Variable Support Like Apollo Client 3

### ISSUE THREAD

**Issue by affansk** [zino-hofmann/graphql-flutter#1351]
does it support reactive variable like apollo client : https://www.apollographql.com/docs/react/local-state/reactive-variables/

**Comment by vincenzopalazzo** [comment]
Yeah, but I do not think this will happen any time soon. but ofc PR are welcome
